nano/node/bootstrap.cpp

```diff
@@ -2253,6 +2258,41 @@ void nano::bootstrap_server::finish_request ()
 	{
 		run_next ();
 	}
+	else
+	{
+		auto this_l (shared_from_this ());
+		std::weak_ptr<nano::bootstrap_server> this_w (shared_from_this ());
+		node->alarm.add (std::chrono::steady_clock::now () + node->config.tcp_server_timeout + std::chrono::seconds (1), [this_w]() {
```

Comment:
do we need this +1 second?

Reply:
I.e. now = 100, tcp_server_timeout = 1. Then we should check after 1 second if 100 + 1 <= now ()
But what if there was some action at timestamp 100.999? Then it still will be timed out at 101 even efter 1 ms left